tardis/tests/test_rpacket.py

```diff
@@ -0,0 +1,148 @@
+import os
+import random
+from ctypes import CDLL, c_double
+
+import pytest
+import numpy as np
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+
+tests = CDLL(test_path)
+
+np.random.seed(1)
+@pytest.fixture
+def doubles():
+	from sys import float_info as floats
+	MAX_INT, MIN_INT = floats[0], floats[3]
+	SIZE = 10
+
+	
```

Comment:
Yes - you are absolutely right with the yield. So we should parameterize our fixtures using the `request.param` method. Have a look here: https://pytest.org/latest/example/parametrize.html